spec/requests/saved_pages_request_spec.rb

```diff
@@ -0,0 +1,148 @@
+RSpec.describe "SavedPages", type: :request do
```

Comment:
Based on [the rspec docs](https://relishapp.com/rspec/rspec-rails/v/3-8/docs/request-specs/request-spec) I think it's more conventional for this file to be `spec/requests/saved_pages_spec.rb` and for this first line to be

```ruby
RSpec.describe "Saved pages" do
```

Reply:
I've unresolved this because it's not been resolved